scripts/linux/pkg/deb/assets/wrapper.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+# This is a wrapper allowing to run RavenDB by regular users *wherever* (directory-wise) they like
```

Comment:
You are making this exposed as `ravendb` command, which is likely not ideal.
maybe `ravendb-local` ?

Reply:
OK, or do we scrap it altogether? Please see my other comment on this one.

Reply:
yes